meta.yaml

```diff
@@ -1,26 +1,25 @@
-{% set version = "2020.7.30.1" %}
+{% set version = "2021.07.0" %}
 
 package:
   name: geocat-viz
   version: {{ version }}
 
 source:
-  url: https://github.com/NCAR/geocat-viz/archive/v{{ version }}.tar.gz
+  git_rev: restructure # https://github.com/NCAR/geocat-viz/archive/v{{ version }}.tar.gz
```

Comment:
I believe we should be using the `main` branch instead of `restructure`.

Reply:
Yup thank you!